Bottlenose/bottlenose.cpp

```diff
@@ -0,0 +1,132 @@
+#include <bottlenose.h>
+//DEFAULTS
+int _currentPin = 8;
+int _msDot = 50;
+int _msDash = 350;
+int _msCharGap = 100;
+int _highDelay = 2;
+int _lowDelay = 3;
+
+int _digits[10][5] ={
```

Comment:
Since these will only ever be 0s and 1s, maybe it would make more sense as a bool.